src/apiCalls.js

```diff
@@ -0,0 +1,21 @@
+const apiPath = 'https://rancid-tomatillos.herokuapp.com/api/v2'
+
+const getAllMovies = () => {
+  return fetch(`${apiPath}/movies`)
+  .then(response => handleResponse(response))
+}
+
+const getSingleMovie = (movieID) => {
+  return fetch(`${apiPath}/movies/${movieID}`)
+  .then(response => handleResponse(response))
+}
+
+const handleResponse = (response) => {
```

Comment:
Great job using `response.ok` to check the response before attempting to parse it.